known-shortcuts/powerpoint.ts

```diff
@@ -1,3 +1,5 @@
+import { AppParams } from "~/types";
+
 const app = {
```

Comment:
The `AppParams` type is being used to cast the `app` object. However, without seeing the definition of `AppParams`, it's important to ensure that all properties of the `app` object match the expected properties of `AppParams`. If there are any mismatches, it could lead to runtime errors or incorrect behavior. Please verify that the `app` object conforms to the `AppParams` type. Additionally, consider adding a type annotation for the `app` constant to ensure type safety and improve code readability.